demo/lightning/checkpoint/multinode/strategies.py

```diff
@@ -7,7 +7,11 @@
 import torch
 import torch.distributed as dist
 from dataflux_core import user_agent
+from dataflux_pytorch.lightning import DatafluxLightningCheckpoint
+from dataflux_pytorch.lightning.gcs_filesystem import (GCSDistributedReader,
+                                                       GCSDistributedWriter)
 from google.cloud import storage
+from lightning.fabric.utilities.types import _PATH
```

Comment:
Dont see _PATH being used here ? Do we need it ?

Reply:
Got the wrong file. Removed the import.